spec/javascript/containers/CuisineShowContainerSpec.js

```diff
@@ -41,25 +44,56 @@ describe('CuisineShowContainer', () => {
       }, 0);
     });
 
-    it('renders a list item for each item returned from the api call', (done) => {
+    it('renders a address list item for each item returned from the api call', (done) => {
       setTimeout(() => {
         expect(wrapper.find('li.rest-address').text()).toBe('52 Shady Lane');
         done();
       }, 0);
     });
 
-    it('renders a list item for each item returned from the api call', (done) => {
+    it('renders a email list item for each item returned from the api call', (done) => {
       setTimeout(() => {
         expect(wrapper.find('li.rest-email').text()).toBe('foo@foo.com');
         done();
       }, 0);
     });
 
-    it('renders a list item for each item returned from the api call', (done) => {
+    it('renders a phone number list item for each item returned from the api call', (done) => {
       setTimeout(() => {
         expect(wrapper.find('h4').text()).toBe('603-867-5309');
         done();
       }, 0);
     });
+
+    it('renders add a restaurant form', (done) => {
+      setTimeout(() => {
+        expect(wrapper.text()).toContain("Restaurant Form");
+        done();
+      }, 0);
+    });
```

Comment:
Test that this container renders the RestaurantFormContainer component, not hardcode text (which would would be tested in RestaurantFormContainerSpec anyways)

Reply:
I changed this container testing per your recommendations to only test for the presence of RestaurantListTiles.

Reply:
The only thing to add is to check that say, the first RestaurantListTile is passed the right props